includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -2554,13 +2554,14 @@ public function get_level3_data_from_order( WC_Order $order ): array {
 		$process_item  = static function( $item ) use ( $currency ) {
 			// Check to see if it is a WC_Order_Item_Product or a WC_Order_Item_Fee.
 			if ( is_a( $item, 'WC_Order_Item_Product' ) ) {
-				$subtotal   = $item->get_subtotal();
-				$product_id = $item->get_variation_id()
+				$subtotal     = $item->get_subtotal();
+				$product_id   = $item->get_variation_id()
 					? $item->get_variation_id()
 					: $item->get_product_id();
+				$product_code = substr( $product_id, 0, 10 );
```

Comment:
Can we make it truncate on 12 chars, please?

Reply:
My apologies. I have fixed this.